scripts/base/frameworks/analyzer/main.zeek

```diff
@@ -88,6 +88,15 @@ export {
 	## Returns: The analyzer name corresponding to the tag.
 	global name: function(tag: Analyzer::Tag) : string;
 
+	## Translates an analyzer type to a string with the analyzer's type.
+	##
+	## Possible values are "protocol", "packet", "file", or "unknown".
+	##
+	## tag: The analyzer tag.
+	##
+	## Returns: The analyzer kind corresponding to the tag.
+	global kind: function(tag: Analyzer::Tag) : string;
```

Comment:
Would it be cleaner to map this to an enum, also in analyzer.log? I've now repeatedly heard that it improves the logs' quality if they automatically carry as much typing/conciseness as possible, and strings looks a bit looser here than necessary.